radicle-cli/src/commands/patch/checkout.rs

```diff
@@ -0,0 +1,61 @@
+use crate::terminal as term;
+use radicle::cob::patch::{self, PatchId};
+use radicle::git;
+use radicle::profile::Profile;
+use radicle::storage::git::Repository;
+
+pub fn run(
+    storage: &Repository,
+    profile: &Profile,
+    git_workdir: &git::raw::Repository,
+    patch_id: &PatchId,
+) -> anyhow::Result<()> {
+    let patchs = patch::Patches::open(profile.public_key, storage)?;
+    let Some(patch) = patchs.get(patch_id)? else {
+        anyhow::bail!("Patch `{patch_id}` not found");
+    };
```

Comment:
I preferer the second solution, but it is good also your, the only problem is that I never use anyhow and I do not know why with your solution there is this error.

```
error[E0277]: the trait bound `Result<_, anyhow::Error>: StdError` is not satisfied
  --> radicle-cli/src/commands/patch/checkout.rs:14:99
   |
14 |     let patch = patchs.get(patch_id)?.ok_or_else(|| anyhow::bail!("Patch `{patch_id}` not found"))?;
   |                                                                                                   ^ the trait `StdError` is not implemented for `Result<_, anyhow::Error>`
   |
   = help: the following other types implement trait `FromResidual<R>`:
             <Result<T, F> as FromResidual<Result<Infallible, E>>>
             <Result<T, F> as FromResidual<Yeet<E>>>
   = note: required for `anyhow::Error` to implement `From<Result<_, anyhow::Error>>`
   = note: required for `Result<(), anyhow::Error>` to implement `FromResidual<Result<Infallible, Result<_, anyhow::Error>>>`

For more information about this error, try `rustc --explain E0277`.
```

Maybe you know the solution already? otherwise tomorrow morning I can drive into it

Reply:
I think you just need `anyhow!` in the ok_or_else block.

Reply:
thank worked thanks